fullcontact/response/audience_response.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module serves the class for wrapping
+FullContact Audience Create and Download
+API Responses.
+"""
+
+from typing import BinaryIO, Union
+
+from .base.base import BaseApiResponse
+
+
+class AudienceCreateResponse(BaseApiResponse):
+    def get_requestId(self):
+        return self.json().get("requestId", None)
+
+
+class AudienceDownloadResponse(BaseApiResponse):
+    SUCCESSFUL_STATUS_CODES = (200,)
```

Comment:
Is this the only place we override the base `SUCCESSFUL_STATUS_CODES`? Maybe that's worth a comment, that in this case we only allow 200's since it downloads a gzip, and we override the parent classes' status codes??

Reply:
Yes. So far, this is the only place where `SUCCESSFUL_STATUS_CODES` has been overridden. This is because in this case, only a 200 is successful. 404 response has a JSON content rather than binary/bytes.